900.version-fixes/f.yaml

```diff
@@ -1,5 +1,7 @@
 # vim: tabstop=39 expandtab softtabstop=39
 
+- { name: f.lux,                       verpat: "[4-9]{1}\\..*",                             incorrect: true } # latest is version 1.2.0.. where does 4.x come from?
+- { name: f.lux,                       ruleset: chocolatey,                                untrusted: true } # sorry chocolatey
```

Comment:
You don't need this, chocolatey is untrusted by default

Reply:
Indeed, actually, it's CRUX 3.4 with the wrong version scheme currently blocking the correct version detection. I will change that in a bit!

Reply:
Are Windows repositories generally marked as untrusted? I ask because of "AppGet"

Reply:
Yes. Not because they are windows, but because they all fake a lot of versions.